.buildkite/scripts/test_rank.sh

```diff
@@ -12,11 +12,11 @@ docker run \
     yarn
 
 # Copy candidate queries into place.
-# TODO: This is a short-term fix to unbreak the build, find a better
-# way to do this.
-ROOT=$(git rev-parse --show-toplevel)
-cp "$ROOT/search/src/test/resources/WorksMultiMatcherQuery.json" "$ROOT/rank/queries/WorksMultiMatcherQuery.json"
-cp "$ROOT/search/src/test/resources/ImagesMultiMatcherQuery.json" "$ROOT/rank/queries/ImagesMultiMatcherQuery.json"
+docker run \
+    --volume $(pwd):/catalogue-api \
+    --workdir /catalogue-api/rank \
+    public.ecr.aws/docker/library/node:14-slim \
```

Comment:
v minor: we've been moving to Node 16 or Node 18 in the front-end repos. Not a big deal, but let's make a note to move this up at some point.